test/cmd/secrets.sh

```diff
@@ -16,40 +16,40 @@ os::cmd::expect_failure_and_text 'oc secrets new foo --type=blah makefile=Makefi
 os::cmd::expect_success 'oc secrets new foo --type=blah makefile=Makefile --confirm'
 os::cmd::expect_success_and_text 'oc get secrets/foo -o jsonpath={.type}' 'blah'
 
-os::cmd::expect_success 'oc secrets new-dockercfg dockercfg --docker-username=sample-user --docker-password=sample-password --docker-email=fake@example.org'
+os::cmd::expect_success 'oc secrets new-dockercfg dockerconfigjson --docker-username=sample-user --docker-password=sample-password --docker-email=fake@example.org'
 # can't use a go template here because the output needs to be base64 decoded.  base64 isn't installed by default in all distros
-os::cmd::expect_success "oc get secrets/dockercfg -o jsonpath='{ .data.\.dockercfg }' | base64 -d > ${HOME}/dockerconfig"
-os::cmd::expect_success 'oc secrets new from-file .dockercfg=${HOME}/dockerconfig'
+os::cmd::expect_success "oc get secrets/dockerconfigjson -o jsonpath='{ .data.\.dockerconfigjson }' | base64 -d > ${HOME}/dockerconfigjson"
+os::cmd::expect_success 'oc secrets new from-file .dockerconfigjson=${HOME}/dockerconfigjson'
```

Comment:
The test is okay, and although it did save to $HOME before this patch, I believe `${BASETMPDIR}` would be a more appropriate location to store `dockerconfigjson`